README.md

```diff
@@ -87,10 +87,31 @@ public virtual ContentReference GoogleAnalyticsSettings { get; set; }
 * Assign the value of the property to the setting you created in the local settings.
 * After you have done this, content under the node that has assigned the local setting should get these settings, while content outside of this structure should get the global settings.
 
+## Customize how settings are resolved
+By default, settings are resolved by searching for properties with name matching the Settings type. If you wish to add a custom way for this, it is possible to add a class implementing __ISettingsResolver__. For example this gives the power to have nested settings.
+
+Implement in a class:
+```csharp
+public class CustomSettingsResolver : ISettingsResolver
+{
+    public int SortOrder => 1;
+    
+    public bool TryResolveSettingFromContent<T>(IContent content, out T setting) where T : SettingsBase
+    {
+        // Add suitable logic to get the correct settings-instance 
+    }
+}
+```
+
+Register it in the di-container:
+```csharp
+services.AddSingleton<ISettingsResolver, CustomSettingsResolver>();
+```
+
```

Comment:
Could you add information on how resolving works when you have multiple resolvers. First resolver that returns an instance wins - right?

Reply:
That is correct. I added a sentence about that